### ISSUE TITLE
Provide runtimes foe beta NVidia drivers

### ISSUE THREAD

**Issue by tannisroot** [flathub/flathub#619]
Since NVidia offers beta drivers with things like new Vulkan extensions, which are sometimes required to run games through DXVK (used in Steam), it would be very nice if Flathub could offer runtimes for these drivers.

**Comment by TingPing** [comment]
Moving forward I think this is the best solution for beta drivers: https://blog.tingping.se/2018/08/26/flatpak-host-extensions.html

At some point there are just too many versions to package.

**Comment by Shished** [comment]
Also there is no runtime for the current beta driver (not vulkan dev version) 410.57.

**Comment by chui2ch** [comment]
To fix the issue on my end I should use FLATPAK_GL_DRIVERS=host? Or do I need to know the path to the drivers on Ubuntu?

**Comment by TingPing** [comment]
@chui2ch You have to actually package them, it won't just work. See my link above for more details.

**Comment by chui2ch** [comment]
@TingPing I went to the link, but to be honest I don't understand what I am supposed to do.

**Comment by TingPing** [comment]
Not really, it is a technical detail for packagers, none of this is intended for users.

**Comment by chui2ch** [comment]
OK, so as a user at this time if I want to use the Beta drivers I should avoid flatpak?

**Comment by TingPing** [comment]
Sadly we don't have a better answer atm so maybe (for GL using applications).

**Comment by chui2ch** [comment]
@TingPing Thanks for taking the time to answer my questions!

**Comment by tannisroot** [comment]
Can runtime be added for 415.18.02? 
Now, I understand that usually you don't package GL runtime for these dev drivers, but this one contains Vulkan transform feedback extension that is required by DXVK (used in Valve's Proton) to make quite a lot of games function properly, and it is not yet present in mainstream drivers.
So now, users have to either stick to mainstream driver and not be able to play their games, or abandon using Flatpak.
So if it's possible, it would be awesome to make an exception here.
